README.md

```diff
@@ -63,14 +63,14 @@ end
 **Loaf** adds `breadcrumb` helper also to the views. Together with controller breadcrumbs, the view breadcrumbs are appended as the last. For instance, to specify view breadcrumb do:
 
 ```ruby
-<% breadcrumb "#{@category.title}", blog_category_path(@category) %>
+<%= breadcrumb @category.title, blog_category_path(@category) %>
 ```
 
 Finally, in your view layout add semantic markup to show breadcrumbs:
 
 ```html
 <ul class='breadcrumbs'>
-  <% breadcrumbs do |name, url, styles| %>
+  <%= breadcrumbs do |name, url, styles| %>
```

Comment:
As I said above, if you put `=` here at the end of the command it'll print something like this:
`[#<Loaf::Crumb:0x007faa491f9f00  ..`